runtime/test-fw/src/mbox_responder.rs

```diff
@@ -62,12 +65,11 @@ fn mbox_responder() {
                 mbox.write_response(digest.as_bytes()).unwrap();
```

Comment:
Instead of using `flip_endianness`, I think you should use `let out: [u8; 48] = digest.into();`. This will do the endianness conversion as from of the `From` trait implementation: https://github.com/chipsalliance/caliptra-sw/blob/23cc74f49ca36e82b3963f2888208244012528ec/drivers/src/array.rs#L110

Reply:
Done